crontab

```diff
@@ -1,4 +1,5 @@
 PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin
 PYTHONPATH=/usr/local/lib/python36.zip:/usr/local/lib/python3.6:/usr/local/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/site-packages:/usr/local/lib/python3.6/site-packages/messybrainz-99.0.0.dev0-py3.6.egg
 GOOGLE_APPLICATION_CREDENTIALS=/code/credentials/bigquery-credentials.json
-* * * * * python /code/listenbrainz/manage.py stats calculate > /var/log/cron.log 2>&1
+* * * * * python /code/listenbrainz/manage.py stats calculate > /var/log/stats.log 2>&1
+00 00 * * 0 python /code/listenbrainz/manage.py dump create > /var/log/dump_create.log 2>&1
```

Comment:
And just to be clear, do data dumps once a week?

Reply:
I'm not sure if this is a MeB standard, but when I do crontabs I always comment each command saying what it is and when it runs